packages/compass-preferences-model/index.d.ts

```diff
@@ -1,8 +1,12 @@
-import AmpersandModel from 'ampersand-model';
-
-export type CompassPreferencesModel = typeof AmpersandModel;
-
-export = CompassPreferencesModel;
+declare class CompassPreferencesModel {
+  fetch: () => void;
+  save: () => void;
+  set: (
+    key: string,
+    value: unknown
+  ) => void;
+};
```

Comment:
How detailed should this declaration be? I think we also use unset, once, on, and probably other methods from Ampersand. If we don't get rid of Ampersand, maybe it makes sense to still export the whole AmpersandModel type? And I also can't find in the code, where is this CompassPreferencesModel being used?

Reply:
exporting the whole ampersand model type definition is very cumbersome. here i am trying to export on the parts we are using and as a result trying to keep it minimal and simple. and as for using this, its being used in `compass-settings/src/utils/user-preferences.ts`